composer.json

```diff
@@ -50,7 +50,7 @@
 
     "autoload-dev": {
         "psr-4": {
-            "Behat\\Tests\\": "tests/"
+            "Behat\\Tests\\": "tests/Behat/Tests"
```

Comment:
The `autoload-dev` config for tests was wrong/broken. This can be contrasted with the `autoload` config:
https://github.com/Behat/Behat/blob/29344fdf8dfaf818ad4a111d3d3121536d1e1ef1/composer.json#L44